tests/utils/test_network.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from smartsim._core.utils.network import find_free_port
+
```

Comment:
just to make sure this gets picked up by the GH CI, can we add
```py
# The tests in this file belong to the group_a group
pytestmark = pytest.mark.group_a
```
or similar lines to this file?

Reply:
100%. Done